tests/test_utils.py

```diff
@@ -18,3 +18,12 @@ def test_snake_layout_add(qtbot, direction, shape):
         qtbot.addWidget(widget)
 
     assert (layout.columnCount(), layout.rowCount()) == shape
+
+
+def test_no_device_lazy_load():
+    from ophyd import Device
+    old_val = Device.lazy_wait_for_connection
+    with no_device_lazy_load():
+        assert Device.lazy_wait_for_connection is False
```

Comment:
Could test with instantiated devices, but I guess it doesn't really matter

Reply:
Addressed via https://github.com/pcdshub/lucid/pull/25/commits/c422343c74d6dff05be887c463b47acaf37918ff